<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DistanceViewDialog</class>
 <widget class="QDialog" name="DistanceViewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View distance</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>361</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Correct value</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>334</width>
      <height>100</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="mashuLabel">
       <property name="text">
        <string>Mashu</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="fromSkillSource">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="toSkillSource">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="offensiveHorseLabel">
       <property name="text">
        <string>-1 Horse</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="fromEquipSource">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="toEquipSource">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="feiyingLabel">
       <property name="text">
        <string>Feiying</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="fromSkillDest">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="toSkillDest">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="defensiveHorseLabel">
       <property name="text">
        <string>+1 Horse</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="fromEquipDest">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="toEquipDest">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="leftDistanceLabel">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>101</y>
     <width>78</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Left distance</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rightDistanceLineEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="leftDistanceLineEdit">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>101</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="rightDistanceLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>84</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Right distance</string>
   </property>
  </widget>
  <widget class="QLabel" name="finalResultLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>72</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Final result</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="finalResultLineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="seatLabel">
      <property name="text">
       <string>Seat</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="fromSeatLineEdit">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="toSeatLineEdit">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="attackRangeLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Attack Range of Source</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="attackRangeLineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="inAttackRangeLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>90</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>In Attack Range</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inAttackRangeLineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>280</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="fromCombobox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="fromLabel">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>21</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>From</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>to</string>
   </property>
  </widget>
  <widget class="QComboBox" name="toCombobox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DistanceViewDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DistanceViewDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
